services/core-web/src/components/mine/Securities/MineSecurityInfo.js

```diff
@@ -139,6 +139,34 @@ export class MineSecurityInfo extends Component {
     });
   };
 
+  openTransferBondModal = (event, bond) => {
+    event.preventDefault();
+    this.props.openModal({
+      props: {
+        title: `Transfer Bond`,
+        onSubmit: this.transferBond,
+        editBond: true,
+        bond,
+        permitGuid: bond.permit_guid,
+        mineGuid: this.props.mineGuid,
+        permits: this.props.permits,
+      },
+      width: "50vw",
+      content: modalConfig.TRANSFER_BOND_MODAL,
+    });
+  };
+
+  transferBond = (previousBondPayload, previousBondGuid, newBondPayload) => {
+    this.props.updateBond(previousBondPayload, previousBondGuid).then(() => {
```

Comment:
I'm not sure if I agree with this implementation - I think we should create a specific 'transfer' endpoint action to deal with this rather than orchestrating it on the front-end. 

With this we don't have a way of capturing whether someone has explicitly transferred a bond, rather than just release and recreate. Plus, there's some error handling concerns as well.